theme_bewise/views/customizations.xml

```diff
@@ -453,4 +453,24 @@
     </xpath>
 </template>
 
+<!-- ======== STRIPED CENTER TOP ======== -->
```

Comment:
The button is almost not visible here
![Screenshot 2024-08-19 at 16 33 14](https://github.com/user-attachments/assets/dc4df19b-e1cf-4b95-a6e5-2003aeea6bc9)

Reply:
Replaced with a cc3 for now. Theme palette definitely needs some fixes